mkosi/__init__.py

```diff
@@ -2438,6 +2438,23 @@ def run_pacman(root: str, pacman_conf: str, packages: Set[str]) -> None:
         run(["gpgconf", "--homedir", os.path.join(root, "etc/pacman.d/gnupg"), "--kill", "all"])
 
 
+def enable_locale(args: CommandLineArguments, root: str) -> None:
```

Comment:
enable_locale() sounds so generic. But this is specific to Debian and a few other distros. Maybe call this function patch_locale_gen() or so, since that's what it does?